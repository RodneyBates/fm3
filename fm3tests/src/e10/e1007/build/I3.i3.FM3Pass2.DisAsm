 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L5(16_5),[1,0])
 ItkBlockDeclListLt(L0(16_0),[10,2])
 ItkBlockDeclListRt(L0(16_0),[10,2])
 ItkInterfaceRt(L5(16_5),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
