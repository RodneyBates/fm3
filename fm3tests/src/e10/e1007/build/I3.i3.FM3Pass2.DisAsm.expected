 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L6(16_6),[1,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L0(16_0),[10,2])
 ItkOpenDeclListRt(L0(16_0),[10,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkInterfaceRt(L6(16_6),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
